configuration/changing-shell/en.md

```diff
@@ -28,6 +28,31 @@ To switch to another shell, first install the appropriate package, followed by t
 
 Example:
 
+For Zsh
+
+``` bash
+chsh -s /bin/zsh
+```
+For Fish
+
+``` bash
+chsh -s /usr/bin/fish
+```
+
```

Comment:
Instead of this, let's just go ahead and have something like:

- For Fish: `chsh -s /usr/bin/fish`
- For Zsh: `chsh -s /bin/zsh`

```
- For Fish: `chsh -s /usr/bin/fish`
- For Zsh: `chsh -s /bin/zsh`
```